sdk/storage/storage-blob/src/generated/src/models/index.ts

```diff
@@ -5883,11 +5883,11 @@ export type ServiceSetPropertiesResponse = ServiceSetPropertiesHeaders & {
    * The underlying HTTP response.
    */
   _response: coreHttp.HttpResponse & {
-      /**
-       * The parsed HTTP response headers.
-       */
-      parsedHeaders: ServiceSetPropertiesHeaders;
-    };
```

Comment:
formatter removes the whitespace? :(

Reply:
Unfortunately :(  But plus side, we can refer back to this PR for these docs when we add them to the swagger file!